crates/yakui-widgets/src/widgets/textbox.rs

```diff
@@ -128,7 +127,7 @@ pub struct TextBoxWidget {
 pub struct TextBoxResponse {
     pub render_text: RenderText,
     pub scroll: Option<cosmic_text::Scroll>,
-    pub text: Option<String>,
+    pub text: String,
```

Comment:
I don't think we can make a change like this: `Option<String>` is important to know whether the text changed or not.

Reply:
Hmmmmmmmmmmmmmmmmmmmmmmm

If that's the reason for an Option<String>, we'd need to somehow know when the text has been modified.